Formula/poke.rb

```diff
@@ -0,0 +1,28 @@
+class Poke < Formula
+  desc "Extensible editor for structured binary data"
+  homepage "https://jemarch.net/poke"
+  url "https://ftp.gnu.org/gnu/poke/poke-3.1.tar.gz"
+  sha256 "f405a6ba810916ec717000b6fe98ef10cbe549bf0a366b3f81e1f176ff8ff13f"
+  license "GPL-3.0-or-later"
+
+  depends_on "bdw-gc"
```

Comment:
Try
```suggestion
  depends_on "pkg-config" => :build
  depends_on "bdw-gc"
```

Reply:
Added, not sure why this worked for me without it though.

Reply:
The build environment has a `pkg-config` shim on macOS that is always on `PATH`, so that effectively means `pkg-config` is always on `PATH` if installed on macOS. There is no equivalent shim on Linux, so `pkg-config` needs to be declared as a dependency there to be in `PATH` during the build.